components/localize/localize-mixin.js

```diff
@@ -1,14 +1,25 @@
 import IntlMessageFormat from 'intl-messageformat/src/main.js';
 window.IntlMessageFormat = IntlMessageFormat;
 
+var assign =
```

Comment:
IE11 eh?  Are we thinking this polyfill for `Object.assign` will get moved somewhere in the future?

Reply:
I hadn't in particular but am open to suggestion there.